lib/brexit_checker/question/option.rb

```diff
@@ -16,7 +16,6 @@ def show?(criteria_keys)
 
   def self.load(params, parent)
```

Comment:
Can we remove it here?
```suggestion
  def self.load(params)
```

and in the `load_all` method?